open-apparel-registry/open-apparel-registry
# Add facility field changes to contributor notification emails

## Overview

Add a list of updated field values to contributor email notifications.

Connects #635

## Testing Instructions

* Claim a facility and approve the claim.
* Browse the facility profile page at http://localhost:6543/claimed/{id}/ and change some fields:
  * Update `Facility Name` to "Changed Name"
  * Update `Phone Number` to "Changed Phone Number"
  * Turn OFF the `Publicly Visible` switch next to `Phone Number`
  * Update `Website` to "change.org"
* Restart `./scripts/server` to ensure that emails are displayed in the console.
* Save the changes. Verify that the email printed to the console includes the following. (Verify both the plain text and HTML versions)

```
The following details have been changed:


  - Facility name: Changed Name
  - Facility website: change.org
```
* Change all the field values on the form, turning on all publicly visible switches.
* Save the form and verify that the email printed to the console shows all the field changes (Verify both the plain text and HTML versions)

```
The following details have been changed:


  - Description: Changed Description 2
  - Facility name: Changed Name 2
  - Address: Changed Address 2
  - Facility phone number: Changed Phone 2
  - Facility website: change-2.org
  - Minimum order quantity: Changed Quantity 2
  - Average lead time: Changed Lead 2
  - Contact person: Changed Contact 2
  - Contact email: changed-email@example.com
  - Office official name: Changed Office 2
  - Office address: Changed Office Address 2
  - Office country code: DM
  - Office phone number: Changed Office Phone 2
  - Parent company: Brand E
```


## Checklist
